src/datasource/dashboards/twinmaker-main-dashboard.json

```diff
@@ -307,22 +431,29 @@
       "type": "grafana-iot-twinmaker-sceneviewer-panel"
     },
     {
-      "datasource": null,
+      "datasource": {
+        "type": "datasource",
+        "uid": "grafana"
+      },
       "gridPos": {
         "h": 12,
         "w": 9,
         "x": 15,
-        "y": 9
+        "y": 16
       },
       "id": 12,
       "options": {
-        "componentName": "${sel_video_comp}",
-        "datasource": "${DS_AWS_IOT TWINMAKER}",
-        "entityId": "${sel_video_entity}",
-        "kvsStreamName": ""
+        "componentName": "sel_video_comp",
+        "datasource": "AWS IoT TwinMaker",
```

Comment:
This should use the same datasource as the others:
```
"datasource": {
            "type": "grafana-iot-twinmaker-datasource",
            "uid": "${DS_AWS_IOT TWINMAKER}"
          },
```

Reply:
Thanks!